### DISCUSSION TITLE
Cannot acess code-server from browser.

### DISCUSSION THREAD

**Discussion by mintgh** [coder/code-server#5123] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/87857426/164599953-bb36908c-e015-455f-9ccc-5da33551fea3.png)

hi i set up my code-server on ubuntu server 20.04, installing was a breeze, though i cannot access the code server on any browsers on the same local network, pls help.

**Comment by lg-nineoneninethreenine** [comment]
You have to expose the port and then access it.

**Reply by mintgh** [reply]
sorry, totally noob question, how to expose the port?

**Reply by mintgh** [reply]
ok so i did try `sudo ufw allow 8080/tcp` and `sudo ufw allow 8080/udp`
didnt seem to work

**Reply by lg-nineoneninethreenine** [reply]
Allow your router to do the same.

**Comment by mintgh** [comment, marked as answer]
nvm i was being silly, apparently i needed to use the local address for the machine that has code-server installed on it, not 0.0.0.0
